vertx-opentelemetry/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,61 @@
+= Vertx OpenTelemetry
+
+Vert.x integration with OpenTelemetry.
+
+[source,$lang]
+----
+{@link examples.OpenTelemetryExamples#ex1}
+----
+
+You can also pass a custom `OpenTelemetry` allowing for greater control
```

Comment:
btw, did you mean "pass a custom `OpenTelemtryOptions` object", something like that?

Reply:
nono, I mean `OpenTelemetry` (wrapped inside `OpenTelemetryOptions`)